lib/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -122,10 +123,11 @@ public function update_item( $request ) {
 			update_post_meta( $data['id'], '_wp_attachment_image_alt', $request['alt_text'] );
 		}
 
-		$response = $this->get_item( array(
-			'id'      => $data['id'],
-			'context' => 'edit',
-		));
+		$attachment_location = rest_url( '/wp/v2/' . $this->get_post_type_base( get_post_type( $data['id'] ) ) . '/' . $data['id'] );
```

Comment:
I understand that. However, to date, I'd always thought `WP_REST_Request` objects *needed* a URL, just as they need a method, because the URL is the canonical resource identifier. Without a URL, the request object could be passed into different methods missing its original context.